chatty-clownfish/bangazon_sprint_3
# Models

## Link to Ticket
Creating foundation for app. Created 12 models through group code

## Steps to Test
git fetch --all
git checkout group-models

python manage.py makemigrations death_star_data
python manage.py migrate

Open DB Browser and ensure all models exist.

## Impacted Areas in Application
models.py

@danielreedcombs 
@buffard 
@johnnathanwood
